apps/communications/contacts/js/contacts.js

```diff
@@ -976,6 +968,19 @@ var Contacts = (function() {
     ice: 'ice-view'
   };
 
+  function loadConfirmDialogs(callback){
```

Comment:
Instead of adding a new function, would it be possible to use `addAsyncScripts` and add this file as the last element of the array?